dora/src/error/msg.rs

```diff
@@ -485,12 +485,16 @@ impl SemErrorAndPos {
     }
 
     pub fn message(&self, vm: &VM) -> String {
-        let file = vm.file(self.file);
-        format!(
-            "error in {} at {}: {}",
-            file.name,
-            self.pos,
-            self.msg.message()
-        )
+        let file = self.filename(vm, self.file);
```

Comment:
Thanks for tackling this! It was indeed very annoying. However I don't think this is the right way to fix this. I have a commit here that should fix this problem: https://github.com/dinfuehr/dora/commit/3e7ccbf364b3c7514261387595ec2c00104a3931

Reply:
Great, thanks, will drop this!